loefsys/events/models/event_registation.py

```diff
@@ -18,7 +18,7 @@ class EventRegistration(models.Model):
     user = models.ForeignKey(
         settings.AUTH_USER_MODEL,
         models.SET_NULL,
-        null=True
+        null=True,
```

Comment:
Comma here is not necessary, there are no additional vars

Reply:
I think the formatting tool does this

Reply:
Believe I added these manually as I saw that sometimes we did and sometimes did not have trailing commas. I then left / added the optional trailing commas to be consistent and in line with the PEP 8 recommendation.